docs-src/stargate-develop/modules/develop/examples/docapi/library/_test.sh

```diff
@@ -0,0 +1,295 @@
+# Data sent to this endpoint is expected to be in JSON lines format (1 document per line)
+# Additionally, you can supply an id-path query parameter to use the value at a particular path in 
+# each document as the document's key in the database, so if all your documents have an id field, 
+# you could set id-path=id and treat the value in id as the document's key. 
+# You can also use any valid path syntax (globs not allowed), e.g. id-path=user.emails.[0].id
+# If id-path is excluded, random UUID's will be assigned to every document, and the response will 
+# have the ID's created corresponding in the same order as the documents were supplied in.
+echo "Create document for multiple readers with batch POST"
+./curl-document-post-mult-readers.sh.tmp | jq -r '.'
+# SET UP THE REST OF THE DATA FOR FURTHER WORK
+echo "Create document for multiple books with batch POST"
+./curl-document-post-mult-books.sh.tmp | jq -r '.'
+echo "Write one reader with an document-id for later use"
+./curl-document-put-one-reader-with-id.sh.tmp | jq -r '.'
+
+# BIG PATCH WARNING: 
+# One thing to note: PATCH expects that the data present at the path is 
+# already a JSON object {}. if you have an array or some scalar value, 
+# those values will be blown away and overwritten
+
+# MODIFY DOCUMENTS
+echo "Replace a document with a PUT using document-id"
+./curl-document-put-replace.sh.tmp | jq -r '.'
+echo "Check that data is included"
+./curl-document-get-one.sh.tmp | jq -r '.'
+echo "PATCH some information into a document using document-id"
+./curl-document-patch.sh.tmp | jq -r '.'
+echo "Check that data is included"
+./curl-document-get-one.sh.tmp | jq -r '.'
+echo "PATCH some more information into a document using document-id"
+./curl-document-patch2.sh.tmp | jq -r '.'
+echo "Check that 2nd data is included"
+./curl-document-get-one.sh.tmp | jq -r '.'
+
+# DECORATIONS FOR POST/PUT/PATCH: TTL, PROFILING
+echo "Patch some more information with profiling using document-id"
+./curl-document-patch3-profiling.sh.tmp | jq -r '.'
+echo "Check that 3rd data is included"
+./curl-document-get-one.sh.tmp | jq -r '.'
+echo "Patch some more information with TTL using document-id"
+./curl-document-put-ttl.sh.tmp | jq -r '.'
+echo "Check that 4th data is included"
+./curl-document-get-one.sh.tmp | jq -r '.'
+
+# WRITE INSERTS WITH DOCUMENT-PATH
+# DOCUMENT-PATH DEFINES WHERE IN A DOCUMENT YOU WANT TO EXECUTE SOMETHING
+# EX: PUT {{base_rest_url}}{{base_doc_api}}/{{namespace}}/collections/{{collection}}/{{bookdocid}}/book/genre
+echo "PUT a document with document-path; replaces current data"
+./curl-document-put-book-with-path.sh.tmp | jq -r '.'
+echo "PATCH a document with document-path - overwrites"
+./curl-document-patch-book-with-path.sh.tmp | jq -r '.'
+
+# MODIFY DOCUMENTS WITH BUILT-IN FUNCTIONS WITH DOCUMENT-PATH
+# FIRST, VIEW THE BUILT-IN FUNCTIONS
+echo "Get built-in functions for a particular namespace"
```

Comment:
`Get builtin functions` endpoint no longer exists in V2, it was removed.

We determined that it will suffices to reject an invalid builtin-function request with a message that "only $push and $pop are supported"